src/Tuura/Boolean.hs

```diff
@@ -1,6 +1,9 @@
+{-# LANGUAGE FlexibleInstances, OverlappingInstances, IncoherentInstances #-}
```

Comment:
I need them for the `Show` instances of `Literal a`. I want different instances for `String` and everything else. Can you suggest a better way?